### ISSUE TITLE
Install failure with playwright

### ISSUE THREAD

**Issue by kenn** [anti-work/shortest#132]
I was following the install guide but failed at running it.

```sh
% npx shortest
Test Execution Error: browserType.launch: Executable doesn't exist at /Users/kenn/Library/Caches/ms-playwright/chromium-1148/chrome-mac/Chromium.app/Contents/MacOS/Chromium
╔═════════════════════════════════════════════════════════════════════════╗
║ Looks like Playwright Test or Playwright was just installed or updated. ║
║ Please run the following command to download new browsers:              ║
║                                                                         ║
║     npx playwright install                                              ║
║                                                                         ║
║ <3 Playwright Team                                                      ║
╚═════════════════════════════════════════════════════════════════════════╝
⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯

 Tests    0 passed (0)
 Duration   0.07s
 Start at   11:27:15 AM

⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯
```

Following the instruction, `playwright` seems to be necessary.

```sh
% npx playwright install
sh: playwright: command not found
```

Manually installed `playwright`, then installed browsers manually.

```sh
% pnpm add -D playwright
% npx playwright install
```

So I think you should mention `playwright` somewhere in the install / quick setup section?

**Comment by m2rads** [comment]
Thanks for bringing it up. Will update the docs.

**Comment by m2rads** [comment]
What is the device you're running this on btw?

**Comment by kenn** [comment]
M3 Mac, latest software

**Comment by kenn** [comment]
Also, is there any reason the install guide does not include `--save-dev` or `-D` option?

Not seeing any point in bundling it for production use.

**Comment by kenn** [comment]
Also, add to `.gitignore`

```sh
.shortest
```

**Comment by m2rads** [comment]
Yes valid points. Will address these. Thanks!

**Comment by m2rads** [comment]
This issue is fixed in v0.1.1. Will close this.

**Comment by m2rads** [comment]
This issue is fixed in v0.1.1. Will close this.
